cranelift/filetests/filetests/isa/x64/simd-conversion-legalize.clif

```diff
@@ -0,0 +1,34 @@
+test legalizer
+set enable_simd
+target x86_64 skylake
```

Comment:
This is currently running the old backend; I think it should be modified to `test compile` and add `feature "experimental_x64"` (see `simd-bitwise-compile.clif`, e.g.).

Reply:
Ok .. Yeah thanks. Will make this change too. It somehow was being acknowledged as testing was failing CI when I had the file tests for conversion to unsigned included, but I was having trouble running it on my machine. Will update.

Reply:
@abrown @bnjbvr .. Actually I am going to just remove this compile file test. It is checking for a very specific sequence of instructions which should not be static (set in stone). It will depend on optimizations or SSE feature flag set and is there anything else that can change register allocation even if the same instructions are used?